scraper.py

```diff
@@ -26,9 +26,12 @@ def get_stock_price():
                 market_data = soup.find('span', attrs={'class' : 'pr'})
                 price_fluctuation = soup.find('span', attrs={'class' : 'chg'})
                 stock_price = market_data.text.strip()
-                print ticker,": $",stock_price, "Price Fluctuation: $",price_fluctuation.text.strip()
+                if price_fluctuation is None:
+                    print ticker, ": $", stock_price
+                else:
```

Comment:
else is reqd otherwise it prints twice for the same stock if the fluctuation is None.